modules/contentbox/modules/contentbox-admin/handlers/authors.cfc

```diff
@@ -332,6 +332,15 @@ component extends="baseHandler"{
         flash.put( "authorID", rc.authorID );
         prc.oAuthor = authorService.get( id=rc.authorID );
 
+        if ( prc.oCurrentAuthor.getAuthorID() != prc.oAuthor.getAuthorID() ) {
+            cbMessagebox.warn( "You cannot enroll another user in two-factor authentication." );
+            flash.keep( "layout,xehInvalidData,xehEnrollmentSuccess,xehEnrollmentSuccessQueryString" )
+            setNextEvent(
```

Comment:
Do a hard `return` so when you do the integration test you can test the relocation.